TrafficCapture/dockerSolution/src/main/docker/docker-compose-multiproxy.yml

```diff
@@ -0,0 +1,154 @@
+version: '3.7'
+services:
+
+  capture-proxy1:
```

Comment:
There wasn't any limitations, but at the time, I didn't have a strong preference either. But now it makes since to have less containers so building locally has less impact on computers performance.
Thank you @sumobrian and @lewijacn for pointing this out.

I just updated the file so the each proxy and elasticsearch node are merged into a single container.